src/components/rangeselector/draw.js

```diff
@@ -180,23 +180,23 @@ function reposition(gd, buttons, opts, axName) {
     var borderWidth = opts.borderwidth;
 
     buttons.each(function() {
-        var button = d3.select(this),
-            text = button.select('.selector-text');
+        var button = d3.select(this);
+        var text = button.select('.selector-text');
 
-        var tHeight = opts.font.size * 1.3,
-            hEff = Math.max(tHeight * svgTextUtils.lineCount(text), 16) + 3;
+        var tHeight = opts.font.size * 1.3;
+        var hEff = Math.max(tHeight * svgTextUtils.lineCount(text), 16) + 3;
 
         opts.height = Math.max(opts.height, hEff);
     });
 
     buttons.each(function() {
-        var button = d3.select(this),
-            rect = button.select('.selector-rect'),
-            text = button.select('.selector-text');
+        var button = d3.select(this);
+        var rect = button.select('.selector-rect');
+        var text = button.select('.selector-text');
 
-        var tWidth = text.node() && Drawing.bBox(text.node()).width,
-            tHeight = opts.font.size * 1.3,
-            tLines = svgTextUtils.lineCount(text);
+        var tWidth = text.node() && Drawing.bBox(text.node()).width;
+        var tHeight = opts.font.size * 1.3;
```

Comment:
Good call. It's used in a few different ways, so I went for `alignment.js` -> https://github.com/plotly/plotly.js/pull/1792/commits/c5fa44b21dac1c3b4039aa4ca2b998e0d2410b30